download-server.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+
+GITHUB_REPO_NAME="streetsidesoftware/vscode-spell-checker"
+
+# download the release
+# more info here - https://gist.github.com/steinwaywhw/a4cd19cda655b8249d908261a62687f8
+
+wget $(curl -s https://api.github.com/repos/${GITHUB_REPO_NAME}/releases/latest | grep 'browser_' | cut -d\" -f4)
```

Comment:
Quoting is a good practice and suggested by linters.

```suggestion
wget "$(curl -s https://api.github.com/repos/${GITHUB_REPO_NAME}/releases/latest | grep 'browser_' | cut -d\" -f4)"
```